utils/a11y_controller.js

```diff
@@ -817,7 +817,7 @@ export default class A11yController {
     }
 
     handleFocus = (event) => {
-        if (this.lastInputEventIsKeyboard && this.windowIsFocused) {
+        if (this.lastInputEventIsKeyboard && this.windowIsFocused && event.target.id !== 'edit_textbox') {
```

Comment:
Why do we need a special case for the edit box here?

Reply:
it placed the `a11y-focussed` class on the element, but there was no easy means of removing that. I do not remember 100%, but it was necessary.

Reply:
Wouldn't we want to add that class to show focus though? I don't know why this component would behave different than any other textarea with the A11yController active

Reply:
Ah, I remember now ... it was needed for the case where the user enters edit mode by using the arrow-up key! ... There was no easy way around this, so I added it here.

Reply:
I still don't understand why this component would have to have special behaviour here though. It looks like this function is only called when the window regains focus, so does that cause the edit box to get stuck with that extra class somehow?


Perhaps we should meet up to chat a bit about this tomorrow since I feel like a few minutes would help me better understand what this is needed for